engine/src/test/resources/org/camunda/bpm/engine/test/cmmn/timer/TimerEventListenerTest.testTimerOccur.cmmn

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="ISO-8859-1" standalone="yes"?>
```

Comment:
It have to look like this:

```xml
<?xml version="1.0" encoding="ISO-8859-1" standalone="yes"?>
<definitions id="_7f0c94c0-2a22-445d-b4b7-4fd181e08248"
             xmlns="http://www.omg.org/spec/CMMN/20151109/MODEL"
             xmlns:camunda="http://camunda.org/schema/1.0/cmmn"
             xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
             targetNamespace="http://www.omg.org/spec/CMMN/20151109/MODEL">
    <case id="case" name="Case">
        <casePlanModel id="CasePlanModel_1">
            <planItem id="PI_Timer_1" definitionRef="Timer_1" />
            <timerEventListener id="Timer_1">
                <extensionElements>
                    <camunda:caseExecutionListener class="org.camunda.bpm.engine.test.cmmn.timer.TimerEventListenerCaseExecutionListener" event="occur" />
                </extensionElements>
                <timerExpression>R3/PT2S</timerExpression>
            </timerEventListener>
        </casePlanModel>
    </case>

</definitions>
```

The `caseExecutionListener` must be added to the `timerEventListener` and you can also provide an event (ie. `event="occur") when the listener should be invoked.